kombuworker/agnostic.py

```diff
@@ -3,11 +3,14 @@
 import json
 import time
 import signal
+
```

Comment:
Is there a reason you want to separate these from the other standard library modules?